src/AuthContext.tsx

```diff
@@ -1,12 +1,27 @@
-import React, { FC, useState, useEffect, useRef, PropsWithChildren, useMemo, useCallback } from 'react';
-import { UserManager, User, SigninRedirectArgs, SignoutRedirectArgs } from 'oidc-client-ts';
+import React, {
+  FC,
+  useState,
+  useEffect,
+  useRef,
+  PropsWithChildren,
+  useMemo,
+  useCallback,
+} from 'react';
+import {
+  UserManager,
+  User,
+  SigninRedirectArgs,
+  SignoutRedirectArgs,
+} from 'oidc-client-ts';
 import {
   Location,
   AuthProviderProps,
   AuthContextProps,
 } from './AuthContextInterface';
 
-export const AuthContext = React.createContext<AuthContextProps | undefined>(undefined);
+export const AuthContext = React.createContext<AuthContextProps | undefined>(
+  undefined,
+);
```

Comment:
I'm assuming it's line breaking this because it exceeds 80 chars, but that feels pretty harsh. Agree/disagree?